src/Libraries/RevitNodes/GeometryConversion/RevitToProtoSolid.cs

```diff
@@ -28,10 +28,22 @@ public static class RevitToProtoSolid
             {
                 srfs.AddRange(face.ToProtoType(false));
             }
-            var converted = Solid.ByJoinedSurfaces(srfs);
+
+            Solid converted = null;
+            try
+            {
+                converted = Solid.ByJoinedSurfaces(srfs);
+            }
+            catch (Exception e)
```

Comment:
@aparajit-pratap 
As discussed, if this function is called to convert a list of geometries, if an exception is let to be thrown for one geometry, other valid geometries won't be returned. Also for the defect that is fixed, the call is happening in an external dll, where you can not log a message to let it to be a warning message.